cares_lib/dynamixel/Servo.py

```diff
@@ -263,15 +286,25 @@ def process_result(self, dxl_comm_result, dxl_error, message="success"):
         logging.debug(f"Dynamixel#{self.motor_id}: {message}")
 
     @staticmethod
-    def step_to_angle(step):
```

Comment:
keep the name step_to_angle - it needs to check the model number and use the right equation

if model == A
 return 
if model == B
return